src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,27 @@
+/* Class created by Priyanka Agarwal. It allows the version of the server
+to be stored and updated. ServerInfo contains the variable Server_Version
+which represents the version, starting with 1.0.0. It can then be updated 
+as needed, whenever the server version changes. The toString method allows 
+the version to be converted from Uuid to a string so that it can be 
+displayed in the Panels, to be use by the Chat.java class.
+ */
+
+package codeu.chat.common;
+import codeu.chat.util.Uuid;
+
+
+public final class ServerInfo {
+
+    public Uuid version;
```

Comment:
Hmm, I'm still not sure why it needs to be public? And minor note with the comments at the top: You no longer have a default version, right? Since you always use the parameterized constructor?